src/install.ts

```diff
@@ -81,9 +81,12 @@ class UI {
     }
   }
 
+  private _pathUpdated: Promise<void>|undefined;
+  private _resolvePathUpdated: (() => void)|undefined;
+
   async promptReload(message: string) {
```

Comment:
I don't think we should ignore `message` entirely. Can we still show it in an info dialog, without waiting for the user to interact with it?

Reply:
I think we can do this:
```ts
  async promptReload(message: string) {
    vscode.window.showInformationMessage(message)
    await this._pathUpdated;
    this._pathUpdated = null;
    vscode.commands.executeCommand('clangd.restart');
  }
```
Without `await`, this won't wait for user prompt and it would serve as a status message `'clangd ${version} is installed'`. Is this what you had in mind?

Reply:
Yep, exactly